packages/relayer/indexer/indexer.go

```diff
@@ -91,7 +92,8 @@ type Indexer struct {
 	bridge     relayer.Bridge
 	destBridge relayer.Bridge
 
-	signalService relayer.SignalService
+	signalService   relayer.ChainDataSignalService
+	signalServiceV2 *v4.SignalService
```

Comment:
I know the version is of the bindings, but it's kind of confusing to use v2 if the bindings are v4, maybe `signalServiceV4` would be better name. WDYT?